src/detect-engine-analyzer.c

```diff
@@ -861,6 +863,47 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_FLOWBITS: {
+                const DetectFlowbitsData *cd = (const DetectFlowbitsData *)smd->ctx;
+
+                jb_open_object(js, "flowbits");
+                switch (cd->cmd) {
+                    /* noalert has been removed and never gets to DumpMatches */
```

Comment:
we haven't like removed anything. It just wasn't supposed to be here so we can remove this comment, I think.